### PR TITLE
Fix some formatting in term storage docs.

### PR DESCRIPTION
**Pull request by christhekeele**

### CODE REVIEW

File: lib/system_registry/term_storage.ex [hunk 1]
```diff
@@ -36,10 +36,10 @@ defmodule SystemRegistry.TermStorage do
...
   A scope to a leaf node to be persisted upon change.
 
-  For Example:
-    Lets say that we want to make sure that the ssid for our wireless
-    network is persisted between reboots.
-    `SystemRegistry.TermStorage.persist([:config, :network_interface, "wlan0", :ssid])``
+  For example, let's say that we want to make sure that the ssid for our wireless
+  network is persisted between reboots:
+
+      SystemRegistry.TermStorage.persist([:config, :network_interface, "wlan0", :ssid])
```
Comment (adkron) [inline lib/system_registry/term_storage.ex]: If we keep the ticks around the code it displays the code in the docs. ie:

`SystemRegistry.TermStorage.persist([:config, :network_interface, "wlan0", :ssid])`

**Comment by christhekeele** [comment]
Currently, it looks like this:

screen shot 2017-09-07 at 11 15 32 am

The quick fix, removing the extra trailing \` on that line, gets a little cramped:

screen shot 2017-09-07 at 11 17 54 am

I liked how this appeared more:

screen shot 2017-09-07 at 11 20 04 am

Whichever way you prefer––just wanted to fix the broken bit. :D

**Comment by adkron** [comment]
I didn't notice the indentation. Yours looks best.

**Comment by christhekeele** [comment]
Should have led with the pictures, sorry.

**Comment by adkron** [comment]
I should have generated the docs to view it. You did a great job. I just got in a hurry when I was looking. :heart:

Amos King
Binary Noggin

**Comment by mobileoverlord** [comment]
Looks good! Thanks for the PR ❤️💙💚💜

**Pull request merged**